tenseal/cpp/tensors/ckksvector.h

```diff
@@ -109,10 +109,8 @@ class CKKSVector {
     /**
      * Matrix multiplication operations.
      **/
-    CKKSVector matmul_plain(const vector<vector<double>>& matrix,
-                            uint n_threads = 0);
-    CKKSVector& matmul_plain_inplace(const vector<vector<double>>& matrix,
-                                     uint n_threads = 0);
+    CKKSVector matmul_plain(const vector<vector<double>>& matrix);
+    CKKSVector& matmul_plain_inplace(const vector<vector<double>>& matrix);
```

Comment:
Even if now we have a thread pool with a certain number of threads, can't we control the number of jobs created for matmul? like the number of threads is n, and someone can specify a number j <= n of jobs? in the API it will look something more like `vec.matmul(matrix, n_jobs=j)`. Do you think this can be useful, I don't know...

Reply:
Sure, that should be easy to add.